src/app/components/Components.jsx

```diff
@@ -79,7 +79,7 @@ class Components extends React.Component {
 
     return (
       <div
-        data-terra-site-content
+        data-terra-dev-site-content
```

Comment:
@emilyrohrbough, will this non-passively break tests? If so we should probably leave it as data-terra-site-content.

Reply:
Given this will be released as a totally new component, I think this change is fine. At this point, I think terra-framework is the only one using it (see [here](https://github.com/cerner/terra-framework/blob/master/wdio.conf.js#L34)) Plus, each repo consuming terra-site will need to update their their [generate-config script](https://github.com/cerner/terra-framework/blob/master/package.json#L52) to use terra-dev-site as well.